.npmignore

```diff
@@ -1,30 +0,0 @@
-build/
```

Comment:
We should keep npmignore with whatever is irrelevant for build.

Reply:
Instead I set `"files": ["dist"]` to `package.json`.